languages/elixir/exercises/concept/captains-log/.docs/hints.md

```diff
@@ -25,13 +24,13 @@
 - There is a [built-in function][to-string] that changes a charlist to a string.
 - The format string of that function contains control sequences.
 - A control sequence starts with `~` and has the pattern `~F.P.PadModC`, where `F` stands for the width of the output, `P` stands for the precision, `Pad` stands for the padding character, `Mod` stands for the control sequence modifier, and `C` is the type of the control sequence.
-- To format a float with a desired precision, a control sequence with the pattern `~P.C` will suffice.
+- To format a float with a desired precision, a control sequence with the pattern `~.PC` will suffice.
 - The control sequence type for floats is `f`.
+- The exact format string you need is `~.1f`.
```

Comment:
To be honest, I couldn't figure out from Erlang docs where the dots are necessary and where not... I want to have the exact value that needs to be used here, in order not to frustrate users with this too much.